spec/controllers/divisions_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'spec_helper'
 
 describe DivisionsController, :type => :controller do
+  describe "#index" do
+    before :each do
+      Division.delete_all
 
+      @division = Division.create(id: 5, name: "hey jude", date: Date.new(2016,12,25),
```

Comment:
I could not use `let` here because it is necessary to have at least one division in the database when doing a GET request in `/divisions`